src/xSTIR/mSTIR/+mSTIR/PLSPrior.m

```diff
@@ -0,0 +1,79 @@
+classdef PLSPrior < mSTIR.Prior
+% Class for PLS priors
+
+% CCP PETMR Synergistic Image Reconstruction Framework (SIRF).
+% Copyright 2015 - 2017 Rutherford Appleton Laboratory STFC.
+% 
+% This is software developed for the Collaborative Computational
+% Project in Positron Emission Tomography and Magnetic Resonance imaging
+% (http://www.ccppetmr.ac.uk/).
+% 
+% Licensed under the Apache License, Version 2.0 (the "License");
+% you may not use this file except in compliance with the License.
+% You may obtain a copy of the License at
+% http://www.apache.org/licenses/LICENSE-2.0
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS,
+% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+% See the License for the specific language governing permissions and
+% limitations under the License.
+
+    methods
+        function self = PLSPrior()
+            self.name = 'PLSPrior';
+            self.handle_ = calllib('mstir', 'mSTIR_newObject', self.name);
+            mUtilities.check_status('PLSPrior:ctor', self.handle_)
+        end
+        function delete(self)
+            mUtilities.delete(self.handle_)
+            self.handle_ = [];
+        end
+        function set_only_2D(self, name)
+%***SIRF*** Sets only 2D.
+            mSTIR.setParameter(self.handle_, 'PLSPrior', 'only_2D', name, 'f')
```

Comment:
the last argument must be 'i'

Reply:
There are a few files that I didn't mean to make changes to, including this one. Almost all the changes should be in `src/Registration`, so I'll just reset this file.

Reply:
@evgueni-ovtchinnikov I was just reseting this, but it seems it doesn't exist on the master. Did you only wrap `PLSPrior` to Python?